documentation/public/user-guide/rust-crate/installation.md

```diff
@@ -1,16 +0,0 @@
-# Installation
-
-The Rust package is published to crates.io as [`slang_solidity`](https://crates.io/crates/slang_solidity) ([docs](https://docs.rs/slang_solidity/latest/slang_solidity/)).
-It can be used both as a regular Rust dependency and as a standalone CLI (installable with Cargo).
-
-You can install the CLI as a cargo binary using:
-
-```bash
-cargo install "slang_solidity_cli"
-```
-
-Or you can add the API as a dependency to your project:
-
-```bash
-cargo add "slang_solidity"
-```
```

Comment:
Perhaps we want to save the Rust documentation somewhere so it can be easily restored when we start publishing the crates again?

Reply:
I removed them because mkdocs will error out if they are not included in navigation.
But looks like they introduced a new [not_in_nav](https://www.mkdocs.org/user-guide/configuration/#not_in_nav) option. I will try using it and if that works!

Reply:
addressed in #1135